Gems/AtomLyIntegration/AtomBridge/Code/Source/AtomDebugDisplayViewportInterface.cpp

```diff
@@ -1025,83 +1025,57 @@ namespace AZ::AtomBridge
+            const AZ::Vector3 worldAxis = ToWorldSpaceVector(axis);
 
             // Draw cylinder part (or just a circle around the middle)
             if (heightStraightSection > FLT_EPSILON)
             {
-                DrawWireCylinder(center, axis, radius, heightStraightSection);
-            }
-            else
-            {
-                float Deg360 = DegToRad(360.0f);
-                CreateArbitraryAxisArc(
-                    lines,
-                    stepAngle,
-                    Deg0,
-                    Deg360,
-                    center,
-                    radiusV3,
-                    axisNormalized
-                    );
+                m_auxGeomPtr->DrawCylinderNoEnds(
+                    worldCenter,
+                    worldAxis,
+                    scale * radius,
+                    scale * heightStraightSection,
+                    m_rendState.m_color,
+                    AZ::RPI::AuxGeomDraw::DrawStyle::Line,
+                    m_rendState.m_depthTest,
+                    m_rendState.m_depthWrite,
+                    m_rendState.m_faceCullMode,
+                    m_rendState.m_viewProjOverrideIndex
+                );
             }
 
-            float Deg90 = DegToRad(90.0f);
-            float Deg180 = DegToRad(180.0f);
-
             AZ::Vector3 ortho1Normalized, ortho2Normalized;
             CalcBasisVectors(axisNormalized, ortho1Normalized, ortho2Normalized);
             AZ::Vector3 centerToTopCircleCenter = axisNormalized * heightStraightSection * 0.5f;
             AZ::Vector3 topCenter = center + centerToTopCircleCenter;
             AZ::Vector3 bottomCenter = center - centerToTopCircleCenter;
 
-            // Draw top cap as two criss-crossing 180deg arcs
-            CreateArbitraryAxisArc(
-                    lines,
-                    stepAngle,
-                    Deg90,
-                    Deg90 + Deg180,
-                    topCenter,
-                    radiusV3,
-                    ortho1Normalized
-                    );
-
-            CreateArbitraryAxisArc(
-                    lines,
-                    stepAngle,
-                    Deg180,
-                    Deg180 + Deg180,
-                    topCenter,
-                    radiusV3,
-                    ortho2Normalized
-                    );
-
-            // Draw bottom cap
-            CreateArbitraryAxisArc(
-                    lines,
-                    stepAngle,
-                    -Deg90,
-                    -Deg90 + Deg180,
-                    bottomCenter,
-                    radiusV3,
-                    ortho1Normalized
-                    );
-
-            CreateArbitraryAxisArc(
-                    lines,
-                    stepAngle,
-                    Deg0,
-                    Deg0 + Deg180,
-                    bottomCenter,
-                    radiusV3,
-                    ortho2Normalized
-                    );
```

Comment:
How do the new hemisphere's look compared to the old ones? Does that need to be checked with the editor team?

Reply:
The old ones looked horrible, literally two arcs crossing at 90 degrees to each other (and they didn't even rotate properly with with cylinder). You can see them in the first screenshot.